misc/cf-support

```diff
@@ -182,6 +182,41 @@ if [ -f $WORKDIR/share/cf-support-nova-hub.sh ]; then
   source $WORKDIR/share/cf-support-nova-hub.sh
 fi
 
+echo "Analyzing CFEngine core dumps"
+_core_log="$tmpdir"/core-dump.log
+cf_core_files=$(find / -name 'core*' -type f -exec file {} \; 2>/dev/null | grep "core file" | grep "cf-" | cut -d' ' -f1 | sed 's/:$//')
+if [ -z "$cf_core_files" ]; then
+  echo "No CFEngine core files found"
+else
+  if ! command -v gdb >/dev/null; then
+    if [ "$non_interactive" -eq 0 ]; then
+      read -r -p "Install gdb to process core dumps? [Y/n]: " response
+    fi
+    response=${response:-y}
```

Comment:
Not sure if the default should be installing new packages.

Reply:
I mean, by default I want to get the information so I think this is the right way to go.